Functions/xASL_init_DefineStudyData.m

```diff
@@ -57,32 +57,35 @@
 
 % ------------------------------------------------------------------------------------------------
 %% Admin
-if ~isfield(x,'name')
-    x.name = ''; 
+if ~isfield(x,'dataset')
+    x.dataset = struct; 
+end
+if ~isfield(x.dataset,'name')
+    x.dataset.name = ''; 

```

Comment:
Check  xternal/SPMmodified/spm_atlas.m, line 77 - is this related?

Reply:
The name in spm_atlas isn't the name of the subject I think. And there we don't work with the exploreasl x struct as far as I can see it, but I never modified that much spm code.

Reply:
You are right - not related...